pkg/config/config_test.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package config
 
+// aaa
```

Comment:
```suggestion
```

Reply:
Invalid comment

Reply:
This is only temporary content. And this PR is WIP

Reply:
> This is only temporary content. And this PR is WIP

I got it. thanks